test/e2e/scheduling/predicates.go

```diff
@@ -367,29 +366,6 @@ var _ = SIGDescribe("SchedulerPredicates [Serial]", func() {
 		verifyResult(cs, 0, 1, ns)
 	})
 
-	It("validates that a pod with an invalid NodeAffinity is rejected", func() {
```

Comment:
rejection of invalid affinity is covered by validation unit test, right? removal of this here LGTM

Reply:
The validation unit test covered empty NodeAffinity; I added two case for invalid key and value :)